locale/en_US/submission.po

```diff
@@ -1448,6 +1472,163 @@ msgstr "Revisions for the submission, {$title}, have been requested."
+msgstr "{$editorName} has removed an empty external review round."
+
+msgid "editor.submission.decision.removeEmptyExternalReviewRound.completed"
+msgstr "Empty external review round has removed successfully"
+
+msgid "editor.submission.decision.removeEmptyExternalReviewRound.completed.description"
+msgstr "Empty external review round for the submission, {$title}, have been removed."
+
+msgid "editor.submission.decision.removeEmptyExternalReviewRound.notifyAuthorsDescription"
+msgstr "Send an email to the authors to let them know that editor has removed an empty external review round from the submission."
+
+msgid "editor.submission.decision.removeEmptyExternalReviewRound.restriction"
+msgstr "Can not delete an external review round if it is the only running review round or has any reviewer assigned to it."
+
+msgid "editor.submission.decision.removeEmptyInternalReviewRound"
+msgstr "Remove Review Round"
+
+msgid "editor.submission.decision.removeEmptyInternalReviewRound.description"
+msgstr "The editor can remove an empty internal review round if no reviewer assigned to that round yet."
+
+msgid "editor.submission.decision.removeEmptyInternalReviewRound.log"
+msgstr "{$editorName} has removed an empty internal review round."
+
+msgid "editor.submission.decision.removeEmptyInternalReviewRound.completed"
+msgstr "Empty internal review round has removed successfully"
+
+msgid "editor.submission.decision.removeEmptyInternalReviewRound.completed.description"
+msgstr "Empty internal review round for the submission, {$title}, have been removed."
+
+msgid "editor.submission.decision.removeEmptyInternalReviewRound.notifyAuthorsDescription"
+msgstr "Send an email to the authors to let them know that editor has removed an empty internal review round from the submission."
+
+msgid "editor.submission.decision.removeEmptyInternalReviewRound.restriction"
+msgstr "Can not delete an internal review round if it is the only running review round or any reviewer assigned to it."
+
+msgid "editor.submission.decision.backToSubmissionFromInternalReviewRound"
+msgstr "Back To Submission"
+
+msgid "editor.submission.decision.backToSubmissionFromInternalReviewRound.description"
+msgstr "The editor can back out from an internal review round to submission stage if no review assignment has completed."
+
+msgid "editor.submission.decision.backToSubmissionFromInternalReviewRound.log"
+msgstr "{$editorName} has backed out from internal review round to submission stage."
+
+msgid "editor.submission.decision.backToSubmissionFromInternalReviewRound.completed"
+msgstr "Back out to submission stage from internal review."
+
+msgid "editor.submission.decision.backToSubmissionFromInternalReviewRound.completed.description"
+msgstr "Has backed out to submission stage from internal review for the submission, {$title}."
+
+msgid "editor.submission.decision.backToSubmissionFromInternalReviewRound.notifyAuthorsDescription"
```

Comment:
I think that all of these `editor.submission.decision.backToSubmissionFromInternalReviewRound` strings can use their equivalent in `editor.submission.decision.backToSubmission`. The user doesn't need the extra details in the text, except the `.log` entries. It will be good to have unique log entries for each decision.

Reply:
Same for the external review round strings.

Reply:
These can probably go, as mentioned in the comment above. But if they stay, all the localized text for internal review needs to go into OMP instead of pkp-lib.

Reply:
I think this comment is still outstanding.

Reply:
Done .